src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommand.cs

```diff
@@ -175,7 +175,12 @@ public void InstallWorkloads(IEnumerable<WorkloadId> workloadIds, bool skipManif
             {
                 // Update currently installed workloads
                 var installedWorkloads = _workloadInstaller.GetWorkloadInstallationRecordRepository().GetInstalledWorkloads(_sdkFeatureBand);
-                workloadIds = workloadIds.Concat(installedWorkloads).Distinct();
+                var previouslyInstalledWorkloads = installedWorkloads.Intersect(workloadIds);
+                if (previouslyInstalledWorkloads.Any())
+                {
+                    _reporter.WriteLine(string.Format(LocalizableStrings.WorkloadAlreadyInstalled, string.Join(" ", previouslyInstalledWorkloads)).Yellow());
+                }
+                workloadIds = workloadIds.Concat(installedWorkloads);
```

Comment:
Does it matter if there are duplicates in the list here (which it looks like will be the case if you specify a workload that is already installed)?  Should we be calling `Distinct()`?
```suggestion
                workloadIds = workloadIds.Concat(installedWorkloads).Distinct();
```